build_tools/packaging/linux/build_package.py

```diff
@@ -87,12 +87,15 @@ def create_deb_package(pkg_name, config: PackageConfig):
 
     if pkg_list is None:
         pkg_list = [pkg_info.get("Package")]
-
     sourcedir_list = []
     for pkg in pkg_list:
         dir_list = filter_components_fromartifactory(pkg, config.gfx_arch)
         sourcedir_list.extend(dir_list)
 
+    print(sourcedir_list)
+    if not sourcedir_list:
+        sys.exit("Empty source directory")
```

Comment:
This logging could use more context:

```suggestion
    print(f"sourcedir_list:\n  {sourcedir_list}")
    if not sourcedir_list:
        sys.exit("Empty sourcedir_list, exiting")
```